downstream/_auxlib/_unpack_hex.py

```diff
@@ -37,16 +45,25 @@ def unpack_hex(hex_str: str, num_items: int) -> np.ndarray:
         ascii_codes = np.fromstring(hex_str.lower(), dtype="S1", sep="").view(
             np.uint8,
         )
+
         digits = ascii_codes - ord("0")
         alphas = ascii_codes - (ord("a") - 10)
-        return np.where(digits < 10, digits, alphas)
+        result = np.where(digits < 10, digits, alphas)
+
+        if byteorder == "little":
```

Comment:
Can you leave a brief comment here? This seems like a nice way to do whatever’s being done but I’m just not sure what that is haha